### DISCUSSION TITLE
part-ii/2018-102/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#301] [Question comments]
Part II 2018 Number Theory: Paper 4, Section I, G 

https://questions.tripos.org/part-ii/2018-102/

**Comment by callumhobbis** [comment]
<details>
<summary>Sketch solution</summary>

The first bit is bookwork (the proof is given at the start of lecture 20) - I think it's sufficient to note that if $\phi=[\overline{a_0,\ldots,a_n}]$, then $\phi=[a_0,\ldots,a_n,\phi]$, and so by the result in lecture 18, $\phi=\frac{\phi p_{n-1}+p_{n-2}}{\phi q_{n-1}+q_{n-2}}$. By this point, I think it's clear we have a quadratic in $\phi$, and $q_{n-1}\neq0$ so $\phi$ is indeed _irrational_.

For the $[7,7,7,\ldots]$ part we just write $x=7+\tfrac1x$ and solve, discarding the negative solution.

The continued fraction expansion is a standard method (example given in lecture 19), and I got $[4,\overline{1,3,1,8}]$. We then find a solution by computing $p_n$ and $q_n$ for $\sqrt{23}$, and I got $(x,y)=(24,5)$ as a solution.

</details>
